addons/website/views/snippets/s_adventure.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_adventure" name="Adventure">
+    <section class="s_adventure pt200 pb200 oe_img_bg" style="background-image: url('/web/image/website.s_adventure_default_image'); background-position: 50% 50%;">
+        <div class="o_we_bg_filter bg-white-50"/>
+        <div class="container">
```

Comment:
Add the `s_allow_columns` class here so that users have more flexiblity to customize the block the way they want.